cmd/snap/cmd_remove_user.go

```diff
@@ -0,0 +1,75 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2020 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package main
+
+import (
+	"fmt"
+
+	"github.com/snapcore/snapd/client"
+	"github.com/snapcore/snapd/i18n"
+
+	"github.com/jessevdk/go-flags"
+)
+
+var shortRemoveUserHelp = i18n.G("Remove a local system user")
+var longRemoveUserHelp = i18n.G(`
+The remove-user command removes a local system user.
```

Comment:
Without digging into our documentation I don't know the semantics of "local system user". Perhaps we could re-word this to say something more like "UNIX account"?

Reply:
I don't have strong opinions, but a unix account does not quite capture it either because it will only remove accounts known to snapd (via login or create). So maybe
```
Remove a user known to snapd`
```
(but that is a bit clunky) ?

Reply:
Two things

* ATM like create-user, this command is hidden
*  local system user is the terminology create-user is using, system-user is also the name of an assertion, I wouldn't change it until we consider making these command visible